src/pages/index.js

```diff
@@ -23,10 +32,18 @@ const Index = ({ location }) => {
       document.body.classList.add('night-mode')
     }
 
+    if (!isMobile) {
+      document.getElementById('___gatsby').style.overflowX = 'visible'
```

Comment:
`document.getElementById('___gatsby')` better to store the element in the variable and check if it exists